index.html

```diff
@@ -889,6 +884,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
+    <section id='external-content'>
```

Comment:
Besides the boilerplate text (Conformance, Terminology, X-Considerations, Acknowledgements, References) the top-level specification elements are the core Fedora capabilities:
1. Resource Management
2. Resource Versioning
3. Binary Resource Fixity
4. Resource Authorization
5. Notifications

With that in mind, I would expect this new section (External LDP-NR Content) to be a sub-element under Resource Management, instead of a new top-level specification element.